subsys/bluetooth/controller/ll_sw/ull_adv_iso.c

```diff
@@ -487,6 +492,17 @@ uint8_t ll_big_terminate(uint8_t big_handle, uint8_t reason)
 		return BT_HCI_ERR_CMD_DISALLOWED;
 	}
 
+	/* Remove ISO data path, keeping data from entering Tx pipeline */
+	num_bis = lll_adv_iso->num_bis;
+	while(num_bis--) {
```

Comment:
Missing space.

```suggestion
	while (num_bis--) {
```

Reply:
Apologies. Working on getting a system with checkpatch.pl up to catch these in future.